src/apps/vpn/ui/screens/settings/ViewSubscriptionManagement/ViewSubscriptionManagement.qml

```diff
@@ -280,17 +280,21 @@ VPNViewBase {
     function getPlanText(currencyCode, amount) {
         const amountDisplay = (amount || 0) / 100;
         const localizedCurrency = VPNLocalizer.localizeCurrency(amountDisplay, currencyCode);
+        const isUsOrCa = VPN.countryCode === "US" || VPN.countryCode === "CA"
```

Comment:
You want to have: if(VPNSubscriptionData.plusTax) ...

Reply:
Plus you want to check the location at the subscription time, not the current one. Because if i travel to Canada it don't start paying taxes with my European subscription. 

This bit must be exploded through Guardian.